src/bridges/chainlink-ccip/AaveCcipGhoBridge.sol

```diff
@@ -0,0 +1,175 @@
+      revert UnsupportedChain();
+    }
+    _;
+  }
+
+  /// @inheritdoc IAaveCcipGhoBridge
+  function transfer(
+    uint64 destinationChainSelector,
+    uint256 amount,
+    PayFeesIn payFeesIn
+  )
+    external
+    payable
+    checkDestination(destinationChainSelector)
+    onlyOwnerOrGuardian
+    returns (bytes32 messageId)
+  {
+    if (amount == 0) {
+      revert InvalidTransferAmount();
+    }
+
+    IERC20(GHO).transferFrom(msg.sender, address(this), amount);
+    IERC20(GHO).approve(ROUTER, amount);
+
+    Client.EVMTokenAmount[] memory tokenAmounts = new Client.EVMTokenAmount[](1);
+    tokenAmounts[0] = Client.EVMTokenAmount({token: GHO, amount: amount});
+
+    Client.EVM2AnyMessage memory message = Client.EVM2AnyMessage({
+      receiver: abi.encode(bridges[destinationChainSelector]),
+      data: '',
+      tokenAmounts: tokenAmounts,
+      extraArgs: '',
+      feeToken: payFeesIn == PayFeesIn.LINK ? LINK : address(0)
+    });
+
+    uint256 fee = IRouterClient(ROUTER).getFee(destinationChainSelector, message);
+
+    if (payFeesIn == PayFeesIn.LINK) {
+      LinkTokenInterface(LINK).transferFrom(msg.sender, address(this), fee);
+      LinkTokenInterface(LINK).approve(ROUTER, fee);
+      messageId = IRouterClient(ROUTER).ccipSend(destinationChainSelector, message);
+    } else {
+      if (msg.value < fee) {
+        revert InsufficientFee();
+      }
+
+      messageId = IRouterClient(ROUTER).ccipSend{value: fee}(destinationChainSelector, message);
+      if (msg.value > fee) {
+        payable(msg.sender).transfer(msg.value - fee);
+      }
+    }
+
+    emit TransferIssued(messageId, destinationChainSelector, amount);
+  }
+
+  /// @inheritdoc IAaveCcipGhoBridge
+  function quoteTransfer(
+    uint64 destinationChainSelector,
+    uint256 amount,
+    PayFeesIn payFeesIn
+  ) external view checkDestination(destinationChainSelector) returns (uint256 fee) {
+    if (amount == 0) {
+      revert InvalidTransferAmount();
+    }
+
+    Client.EVMTokenAmount[] memory tokenAmounts = new Client.EVMTokenAmount[](1);
+    tokenAmounts[0] = Client.EVMTokenAmount({token: GHO, amount: amount});
+
+    Client.EVM2AnyMessage memory message = Client.EVM2AnyMessage({
+      receiver: abi.encode(bridges[destinationChainSelector]),
+      data: '',
+      tokenAmounts: tokenAmounts,
+      extraArgs: '',
+      feeToken: payFeesIn == PayFeesIn.LINK ? LINK : address(0)
+    });
+
+    fee = IRouterClient(ROUTER).getFee(destinationChainSelector, message);
+  }
+
+  /// @inheritdoc CCIPReceiver
+  function _ccipReceive(Client.Any2EVMMessage memory message) internal override {
+    bytes32 messageId = message.messageId;
+    uint64 sourceChainSelector = message.sourceChainSelector;
+    address bridge = abi.decode(message.sender, (address));
+    Client.EVMTokenAmount[] memory tokenAmounts = message.destTokenAmounts;
+
+    if (bridge != bridges[sourceChainSelector] || tokenAmounts.length != 1) {
```

Comment:
```suggestion
    if (tokenAmounts[0].token != GHO || tokenAmounts[0].amount == 0) {
```

It doesn't really matter who sent the GHO to AAVE in a new chain. Enforcing the sender being the corresponding bridge on other network introduces an unnecessary restriction.

The messageId will give all the information needed. All that needs to be checked is if GHO has been sent or not.